src/components/RoomAvatar.js

```diff
@@ -0,0 +1,30 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import ActiveRoomAvatar from '../../assets/images/avatars/room.svg';
+import DeletedRoomAvatar from '../../assets/images/avatars/deleted-room.svg';
+
+const propTypes = {
+    /** Extra styles to pass to Image */
+    avatarStyle: PropTypes.arrayOf(PropTypes.object),
```

Comment:
This is going to throw a 
```
Warning: Failed prop type: Invalid prop `avatarStyle` of type `object` supplied to `RoomAvatar`, expected an array.
```
in the js console because  of [the `StyleSheet.flatten` in this file](https://github.com/Expensify/Expensify.cash/blob/0f58b1c818c0c81d578c812c1ef0a7ccaf8501a3/src/components/Avatar.js#L49).

You could either change this to just `PropTypes.object` or move the `StyleSheet.flatten` to [L22 and L23 of this file](https://github.com/Expensify/Expensify.cash/blob/199afa1867c549e01e03b07a001bad1fcc058dbc/src/components/RoomAvatar.js#L22-L23). I think it's better to move the StyleSheet.flatten because technically it's just needed for the SVG type of objects.